src/adapters/catalonia.js

```diff
@@ -107,4 +107,95 @@ const getLatestMeasurements = function (measurements) {
     }
   });
   return Object.values(latestMeasurements);
-}
\ No newline at end of file
+}
+// remove stations deemed to be duplicates of EEA stations
+function filterDuplicates(measurements, criteria) {
+  return measurements.filter(measurement => {
```

Comment:
are we checking each parameter here to see if it is duplicated? seems like it would be best to just wholesale exclude an entire station if anything is reported. The provider gets linked at the sensor_node level not at the sensor level. i.e. we cant get PM2.5 from provider A and PM10 from provider B and be able to show that distinction

Reply:
right but youre checking at the measurements level using the `measurement.parameter === criterion` seems like thats an unecessary check if we can just check at the station level

Reply:
ok done. just to be clear going for I should only check for duplicates using coordinates, not coordinates + parameters?